server/models/order.js

```diff
@@ -20,10 +20,10 @@ module.exports = (sequelize, DataTypes) => {
       foreignKey: 'customerId',
     })
     Order.belongsTo(models.Store, {
-      foreignKey: 'deliveryStoreId',
+      foreignKey: 'storeId',
     })
-    Order.belongsTo(models.Store, {
-      foreignKey: 'pickupStoreId',
+    Order.hasOne(models.OrderStatus, {
```

Comment:
seems like order.belongsTo(orderstatus) was correct
coz the foreign key orderStatusId is put inside Orders table